sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/TextPollBlockViewMeasurer.kt

```diff
@@ -0,0 +1,44 @@
+package io.rover.sdk.ui.blocks.poll
+
+import android.graphics.Paint
+import io.rover.sdk.data.domain.Color
+import io.rover.sdk.data.domain.Font
+import io.rover.sdk.data.domain.TextAlignment
+import io.rover.sdk.data.domain.TextPollBlock
+import io.rover.sdk.platform.mapToFont
+import io.rover.sdk.services.MeasurementService
+import io.rover.sdk.ui.RectF
+import io.rover.sdk.ui.asAndroidColor
+import io.rover.sdk.ui.blocks.concerns.layout.Measurable
+import io.rover.sdk.ui.blocks.concerns.text.FontAppearance
+
+internal class TextPollBlockViewMeasurer(
+    private val textPollBlock: TextPollBlock,
+    private val measurementService: MeasurementService
+) : Measurable {
```

Comment:
Haha this turns out to be a funny thing.

I can see why you did it: you don't have a view model for the  contained "text poll" itself, separate from the containing block.

While I don't know if this works for you, you could try making a `TextPollViewModel`. I think you're going to want behaviour in there ultimately (particularly for doing the flip-to-result behaviour), so I think the measurement logic ought to go in there.

What do you think?